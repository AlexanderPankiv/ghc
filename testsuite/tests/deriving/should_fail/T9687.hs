{-# OPTIONS_GHC -fwarn-deriving-typeable -Werror #-}
module T9687 where
import Data.Typeable

instance Typeable (a,b,c,d,e,f,g,h)
