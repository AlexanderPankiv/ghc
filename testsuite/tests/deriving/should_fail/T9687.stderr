
T9687.hs:5:10:
    Class ‘Typeable’ does not support user-specified instances
